runtime/common/src/crowdloan/mod.rs

```diff
@@ -434,8 +434,6 @@ pub mod pallet {
 			);
 
 			NextFundIndex::<T>::put(new_fund_index);
-			// Add a lock to the para so that the configuration cannot be changed.
-			T::Registrar::apply_lock(index);
```

Comment:
So there can be changes to the parachain while there is an auction?

Reply:
While there is a crowdloan, yes. You are basically bidding on whatever the controller of the paraid to do. This is similar to many teams uploading a "fake runtime", which is then upgraded after the win to a real runtime.